packages/account/src/Sections/Profile/PhoneVerification/otp-verification.tsx

```diff
@@ -38,19 +43,34 @@ const ConfirmYourEmail = observer(() => {
     return (
         <PhoneVerificationCard is_small_card>
             <Text bold>
-                <Localize i18n_default_text="Confirm it's you" />
+                {should_show_phone_number_otp ? (
+                    <Localize i18n_default_text='Verify your number' />
+                ) : (
+                    <Localize i18n_default_text="Confirm it's you" />
+                )}
             </Text>
             <div className='phone-verification__card--email-verification-content'>
-                <Text size='sm'>
-                    <Localize
-                        i18n_default_text="We've sent a verification code to <0>{{users_email}}</0>."
-                        values={{ users_email: email }}
-                        components={[<strong key={0} />]}
-                    />
-                </Text>
-                <Text size='sm'>
-                    <Localize i18n_default_text='Enter the code or click the link in the email to verify that the account belongs to you.' />
-                </Text>
+                {should_show_phone_number_otp ? (
```

Comment:
Based on BE discussion, we will have an object in phone_number verification which will have status of both email and phone verification. If so cant we just rely on BE statuses

Reply:
Yeapp for now the BE API is not ready yet, I'll go on this approach just to mock the scene of it. Will add a TODO here 🙏